### PR TITLE
Add kwargs space in config for database/dataset combo

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
## What is the current behavior?
We need a space where we can pass the kwargs specif to a database/dataset combination:
if we have a dataset:
`s3://astro-sdk/benchmark/trimmed/covid_overview/covid_overview_10kb.csv`

We need different kwargs passed to native method for different db
1. Redshift
```
{
         native_support_kwargs: {
            "IGNOREHEADER": 1,
             "REGION": "us-west-2",
             "IAM_ROLE": REDSHIFT_NATIVE_LOAD_IAM_ROLE_ARN".
         },
         "enable_native_fallback": false
    }
```

2. Bigquery

```
{
       native_support_kwargs: {
             "skip_leading_rows": 1,
       },
      "enable_native_fallback": false
}
```
related: #972

## What is the new behavior?
Added a field in config to have this db/dataset-specific config.

## Does this introduce a breaking change?
Nope

### CODE REVIEW

**Review by pankajastro — COMMENTED** [review]
Since changes are related to benchmarking I feel we should run the benchmarking job in CI to make sure everything is on the expected line

**Comment by sunank200** [comment]
Should we run the CI job once in this branch @utkarsharma2 ?

**Comment by utkarsharma2** [comment]
@pankajastro @sunank200 S3 to Biquery worked on my local -
Screenshot 2022-10-03 at 3 06 23 PM

File: python-sdk/tests/benchmark/dags/evaluate_load_file.py [hunk 1]
```diff
@@ -73,9 +75,74 @@ def create_dag(database_name, table_args, dataset, kwargs):
...
+        #         }
+        #     2.  Bigquery
+        #         {
+        #             native_support_kwargs: {
+        #                 "skip_leading_rows": 1,
+        #             },
+        #             "enable_native_fallback": false
+        #         }
+        # Note - If there is a config present in both local and global config.
+        # Local config will override the global one.
```
Comment (pankajastro) [inline python-sdk/tests/benchmark/dags/evaluate_load_file.py:141]: shall we keep this example in config.json or docstring, WDYT?

**Review by pankajastro — APPROVED** [review]

File: python-sdk/tests/benchmark/dags/evaluate_load_file.py (hunk 1 above)
Comment (utkarsharma2) [reply python-sdk/tests/benchmark/dags/evaluate_load_file.py]: Moved it to doc string.

**Review by sunank200 — APPROVED** [review]

**Pull request merged**